### PR TITLE
Added signing NuGet packages

### PR DESCRIPTION
**Pull request by jainaashish**
Updated our build definition to sign our nupkgs through MicroBuild signing plugin.

@rrelyea

### CODE REVIEW

File: build/vsts_build.yaml [hunk 1]
```diff
@@ -178,6 +179,14 @@ phases:
       configuration: "$(BuildConfiguration)"
       msbuildArguments: "/t:Pack /p:BuildRTM=$(BuildRTM) /p:ExcludeTestProjects=$(BuildRTM) /p:BuildNumber=$(BuildNumber)"
 
+  - task: MSBuild@1
+    displayName: "Sign Nupkgs"
+    inputs:
+      solution: "build\\sign.proj"
+      msbuildVersion: "15.0"
+      configuration: "$(BuildConfiguration)"
+      msbuildArguments: "/t:AfterBuild /p:SignNupkgs=true"
```
Comment (rohit21agrawal) [inline build/vsts_build.yaml]: you shouldn't have to do this since this is the same step that signs the DLLs

File: build/vsts_build.yaml [hunk 2]
```diff
@@ -83,6 +83,7 @@ phases:
   - task: MicroBuildSigningPlugin@1
     inputs:
       signType: "$(SigningType)"
+      esrpSigning: "true"
```
Comment (rohit21agrawal) [inline build/vsts_build.yaml:86]: how does this work for test signing?

**Review by rohit21agrawal — APPROVED** [review]
just a small question about test signing, otherwise looks good . :shipit:

File: build/vsts_build.yaml (hunk 2 above)
Comment (jainaashish) [reply build/vsts_build.yaml:86]: this is ignored for test sign and only matter when it's real sign.

Comment (rohit21agrawal) [reply build/vsts_build.yaml:86]: And are the packages signed in place? Or are they copied to some location?

**Review by mishra14 — APPROVED** [review]

**Review by dtivel — APPROVED** [review]

**Pull request merged**